# 描述电主轴启停与转速消息

bool isCanLink
bool isSpindleStart
float32 spindleSpeed
float32 spindleSpeedFB
